lib/service.middleware/apiviewRoutes.js

```diff
@@ -273,6 +273,11 @@ class ApiViewRoutes extends ServiceMiddleware {
 
   // TODO: to many input, need some work
   _handlerWrapper (handlerFunc, resolved, skipOnError, service, controller, orgOutput) {
+    // do nothing if a response has already been sent
+    if (resolved['$rawResponse'].finished) {
```

Comment:
What are your thoughts on this?  It solves the issue I was running into where I could not prevent the controller methods from running when an early response occurred with a status code < 400.